com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.Debug.cs

```diff
@@ -132,16 +141,14 @@ void RegisterDebug()
             }
 #endif
 
-            if (debugDisplay.drawCells || debugDisplay.drawBricks)
-            {
-                subdivContainer.children.Add(new DebugUI.FloatField { displayName = "Culling Distance", getter = () => debugDisplay.subdivisionViewCullingDistance, setter = value => debugDisplay.subdivisionViewCullingDistance = value, min = () => 0.0f });
-            }
+            subdivContainer.children.Add(new DebugUI.FloatField { displayName = "Culling Distance", getter = () => debugDisplay.subdivisionViewCullingDistance, setter = value => debugDisplay.subdivisionViewCullingDistance = value, min = () => 0.0f });
```

Comment:
Why removing the check here?

Reply:
Because I want to be able to change the culling distance _before_ enabling any debug drawing to avoid my machine grinding to a halt when it defaults to drawing everything.